Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:boron
+
+WORKDIR /eslint-config-fusion
+
+COPY package.json yarn.lock /eslint-config-fusion/
+RUN yarn
```

Comment:
Thanks, I'll look into this. We can definitely create that user, but since we're using a plugin[1] for the docker commands, I'm not sure if we have control to specify the user right now? We can always open up an issue though if it's important.

[1] https://github.com/buildkite-plugins/docker-compose-buildkite-plugin

Reply:
Also found out that that we have user namespacing enabled: https://docs.docker.com/engine/security/userns-remap/

Though if this is best practice for local development, we can still look into it.

Reply:
The node base image should include a `node` user already, so it should be as simple as a `USER node` step prior to executing any `node` process(es).

Reply:
I will see what we can do here (node steps executed in other pipeline steps). I do think we should be safe though due to the user namespacing.

> and so forth. If a process attempts to escalate privilege outside of the namespace, the process is running as an unprivileged high-number UID on the host, which does not even map to a real user. This means the process has no privileges on the host system at all.